script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2.txt

```diff
@@ -10,8 +10,8 @@ Even though you came all the way here to kill this 'Castle Lord'?
 Any time vampires meet, mortal combat will ensue. Cases like this where vampires part ways after meeting each other are incredibly rare."
 "...Hmm. But it seemed like he knew a lot about you."
 <Page4>
-"We've never met. I myself am very well-known amongst vampires, so he just happens to know who I am. The same goes for the other vampires, none of us are particularly well-acquainted.
-But we do know each other's names and identities. It's only natural that the more power we possess and the longer our histories, the more our name becomes known, right?"
+"We've never met. I myself am very well-known amongst vampires, so he just happens to know who I am. The same goes for the other vampires, none of us are particularly well-acquainted.//to be rechecked
```

Comment:
I'm well known amongst vampires
(no hyphen is used when well known follows the noun it describes)

Reply:
I didn't know that rule, but you're right that I think it looks more correct without the hyphen. I also think "among" reads better. I'm losing my god damn mind with among vs amongst